elastic/datadog_checks/elastic/metrics.py

```diff
@@ -802,6 +813,9 @@ def stats_for_version(version, jvm_rate=False):
         metrics.update(ADDITIONAL_METRICS_PRE_7_0_0)
     if version < [8, 0, 0]:
         metrics.update(ADDITIONAL_METRIC_PRE_8_0_0)
+    if version < [9, 0, 0]:
```

Comment:
What makes this conditional necessary? afaik, there is no version 9 out yet, so we don't know if these metrics are disabled there.

Related to this, have you considered adding these metrics to `VERSIONS_THAT_ADD_METRICS`?